ksqldb-functional-tests/src/test/resources/test-runner/correct/join/output.json

```diff
@@ -1,13 +1,7 @@
 {
   "outputs": [
-    {"topic": "LEFT_OUTER_JOIN", "key": 0, "value": {"NAME": "zero", "VALUE": 0, "F1": null, "F2": null}, "timestamp": 0},
```

Comment:
@spena can you comment on these changes? The testing tool no longer emits join results for left source inputs that don't have any matches on the left side, even though this is a stream-stream left join. This does not seem correct to me?

Reply:
If you want to play around with this yourself, `KsqlTestingToolTest#shouldRunCorrectTests()` fails without these changes.

Reply:
The results are correct. The new stream-stream left join semantics do not emit results that do not have a join until the window + grace period has expired or a join is found on the right topic. The output of this test is emitting only the joined records. There are changes we must do to test the left/outer joins, though. But that's something we can do in a follow-up in order to unblock master.

Reply:
What's the default grace period? Also, what changes are needed in order to begin testing left and outer stream-stream joins again? The fact that we only have test coverage for inner stream-stream joins right now is quite a large test gap. (Agreed that we should merge this PR first to unblock master, though.)